### PR TITLE
Adding functionality

### PR DESCRIPTION
**Pull request by Trust-007**
# Implemented CRUD on Todo-List app
- Remove all hardcoded items from the tasks array.
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.
- Reordered index when item is removed or added
- Save changes to localstorage

### CODE REVIEW

**Review by sajid-munawar — CHANGES_REQUESTED** [review]
Hi there, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project that does not have linters, Kindly make sure your linters are all set d and passed before submitting the PR.

## Stlylint AutoFix
- [ ] if you have a lot of CSS line errors, I kindly suggest you fix them with `npx stylelint "**/*.{css,scss}"`, but please get used to fixing the errors line by line also because it'll help you learn best practices.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

File: src/index.js [hunk 1]
```diff
@@ -5,63 +5,50 @@ import '@fortawesome/fontawesome-free/js/solid';
...
+    const listDescription = document.createElement('div');
+    listDescription.innerHTML = item.description;
+    mainList.appendChild(listDescription);
+    list.appendChild(mainList);
+    const menu = document.createElement('img');
+    menu.className = 'info';
+    //const icon = document.createElement('p');
+    menu.src = menuIcon;
+    //icon.className = 'fa-solid fa-ellipsis-vertical';
+   // option.appendChild(icon);
```
Comment (levy002) [inline src/index.js]: - [x] Kindly make sure to remove the commented code in your codebase.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Trust-007   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [x] Kindly make sure to implement `edit` functionality so the user can be able to edit the task.
- [x] Kindly make sure to fix your app because when the user clicks on the `checkbox` button the description disappears.

    ### This is how your app behaves now
   chrome-capture-2022-10-3


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] I highly recommend you change your `id` for the task, kindly make it more professional and unique. You can use [this package](https://www.npmjs.com/package/uuid) to generate the unique id for your tasks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Trust-007 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] Implemented all required changes from the previous code reviewer :white_check_mark: 

Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**